### PR TITLE
changed nonstandardpath finding tests to use home dir

### PR DESCRIPTION
**Pull request by rwcarlsen**
no more sudo needed.

### CODE REVIEW

File: src/Testing/DynamicLoadingTests.cpp [hunk 1]
```diff
@@ -24,7 +24,7 @@
   // set up
   string name = "otherfac";
   string lib_name = "lib" + name + DynamicModule::suffix();
-  fs::path path = fs::path(Env::getInstallPath()) / fs::path("test_path") / fs::path(lib_name);
+  fs::path path = fs::path(getenv("HOME")) / fs::path("test_path") / fs::path(lib_name);
```
Comment (gidden) [inline src/Testing/DynamicLoadingTests.cpp]: Could we make this in $HOME/.tmp or something similar?  If not, and some exception is thrown, we might not delete it.

File: src/Testing/EnvironmentTests.cpp [hunk 2]
```diff
@@ -21,7 +21,7 @@
 //- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 TEST(EnvironmentTests, FindNonStandardPath) {
   fs::path fname("libfoo.so");
-  fs::path dir = fs::path(Env::getInstallPath()) / fs::path("test_path");
+  fs::path dir = fs::path(getenv("HOME")) / fs::path("test_path");
```
Comment (gidden) [inline src/Testing/EnvironmentTests.cpp]: Same comment as L:27 as DynamicLoadingTests.cpp

**Comment by rwcarlsen** [comment]
done.

**Comment by rwcarlsen** [comment]
be sure to do a merge --ff-only into your branch.

**Pull request merged**